### ISSUE TITLE
Cannot read property 'hasOwnProperty' of undefined

### ISSUE THREAD

**Issue by jbednar** [pyviz/panel#77]
```
import param, panel as pn
pn.extension()

class A(param.Parameterized):
    a = param.Number(7, bounds=(0,10))
    b = param.Boolean(True)
    c = param.String("Yes")

class B(param.Parameterized):
    p = param.ObjectSelector(A())

    @param.depends("p.param", watch=True)
    def fn(self):
        print("fn")
        pass
    
b=B()
pn.Row(b)
```
![image](https://user-images.githubusercontent.com/1695496/46417386-24b31680-c6ef-11e8-8b9c-7326e31d8546.png)

```
b.p.a=8
```

![image](https://user-images.githubusercontent.com/1695496/46417425-414f4e80-c6ef-11e8-82d8-47b1dbee6216.png)

**Comment by philippjfr** [comment]
Thanks, I've seen similar errors a few times but haven't been able to figure out what causes them. This example also appears to work fine in JupyterLab.

**Comment by jbednar** [comment]
I just checked again in classic Jupyter in my envt, and it's still reproducible there, so maybe try in classic?

**Comment by philippjfr** [comment]
Can't reproduce this anymore, probably fixed in bokeh 1.1.0 or something.
